src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -203,6 +218,40 @@ sample({
   target: $accounts,
 });
 
+sample({
+  clock: walletRemoved,
+  source: $accounts,
+  fn: (accounts, wallet) => ({ accounts: accounts.filter((a) => a.walletId === wallet.id), wallet }),
+  target: removeWalletFx,
+});
+
+sample({
+  clock: removeWalletFx.doneData,
+  source: $wallets,
+  fn: (wallets, wallet) => wallets.filter((w) => w.id !== wallet.id),
+  target: $wallets,
+});
+
+sample({
+  clock: removeWalletFx.doneData,
+  source: $accounts,
+  fn: (accounts, wallet) => accounts.filter((a) => a.walletId !== wallet.id),
+  target: $accounts,
+});
+
+sample({
+  clock: removeWalletFx.doneData,
+  source: {
+    activeWallet: $activeWallet,
+    wallets: $wallets,
```

Comment:
I think we could do even better
```javascript
sample({
  clock: walletSelected,
  source: $activeWallet,
  filter: (wallet, nextId) => wallet?.id !== nextId,
  fn: (wallet, nextId) => ({ prevId: wallet?.id, nextId }),
  target: walletSelectedFx,
});
```
Skip wallet selection for already active wallet inside `walletSelected` event
So you should do 
```javascript
forward({ from: removeWalletFx.doneData, to: walletSelected });
```